app/services/idv/phone_step.rb

```diff
@@ -106,7 +106,7 @@ def update_idv_session
       idv_session.address_verification_mechanism = :phone
       idv_session.applicant = applicant
       idv_session.vendor_phone_confirmation = true
-      idv_session.user_phone_confirmation = phone_matches_user_phone?
+      idv_session.user_phone_confirmation = false
```

Comment:
This is the crux and only needed change of this PR, everything else is to support changing the specs